git-refresher/syncing-changes-to-remote.qmd

```diff
@@ -1,18 +1,55 @@
---- 
+---
 title: "Syncing to the remote"
 ---
 
-...
+This section will guide you through syncing your local repo to the remote one.
+
+Any change made so far is only being applied to the local copy of the repository (i.e. the copy on your laptop). To apply your changes to the remote repository (i.e. on GitHub or Dev Ops), you need to “push” the changes.
+
+Having [recorded your changes](recording-changes.html), follow the instructions below to sync to the remote.
 
 ::: panel-tabset
 ## Git Bash
 
+The first time you update your remote repo with the changes you committed to your local one (aka push the changes), use the command below:
+
+`git push --set-upstream <github_id>_main`
```

Comment:
As we found together, this needs to be 'git push --set-upstream origin <githubid>_main'